themes/theme-b2c-tailwind/src/molecules/banner.css

```
@@ -0,0 +1,46 @@
+[data-store-banner-image] {
+  @apply w-full;
+  height: 250px;
```

Why did you choose to use `height: 250px` instead of something like:
```suggestion
  @apply w-full h-64;
```
?

---

My bad! I was testing and forgot to update this! Thankss!